stream-chat-android-ui-components/src/main/res/values/attrs.xml

```diff
@@ -1,208 +1,223 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <declare-styleable name="StreamAvatarView">
-        <attr name="streamAvatarBorderWidth" />
-        <attr name="streamAvatarBorderColor" />
-        <attr name="streamAvatarTextSize" />
-        <attr name="streamAvatarTextColor" />
-        <attr name="streamAvatarTextFont" />
-        <attr name="streamAvatarTextFontAssets" />
-        <attr name="streamAvatarTextStyle" />
-        <attr name="streamAvatarOnlineIndicatorEnabled" format="boolean" />
+    <attr name="streamUiAvatarBorderWidth" format="dimension|reference" />
+    <attr name="streamUiAvatarBorderColor" format="color|reference" />
+    <attr name="streamUiAvatarTextSize" format="dimension|reference" />
+    <attr name="streamUiAvatarTextColor" format="color|reference" />
+    <attr name="streamUiAvatarTextFont" format="reference" />
+    <attr name="streamUiAvatarTextFontAssets" format="string" />
+    <attr name="streamUiAvatarTextStyle">
+        <flag name="normal" value="0" />
+        <flag name="bold" value="1" />
+        <flag name="italic" value="2" />
+    </attr>
+
+    <attr name="streamUiAvatarWidth" format="dimension|reference" />
+    <attr name="streamUiAvatarHeight" format="dimension|reference" />
```

Comment:
These are newly added, so that we use these instead of the similar attributes declared in the `stream-chat-android` module